### DISCUSSION TITLE
popdown rendering in markdown for posts (like the table of contents)

### DISCUSSION THREAD

**Discussion by cs-mshah** [adityatelange/hugo-PaperMod#658] [Q&A] [answered]
Why does the markdown not render the popdown like feature.  
I want to render:  
```
<details>
<summary>my popdown</summary>
some codefenced code.
</details>
```  

which behaves like this:  
<details>
<summary>my popdown</summary>
some codefenced code.
</details>  

This is very useful when we want to hide large chunks of code and display only when the person reading wants to open a particular block. The popdown should be possible with any kind of element - code, image, video, quotes, lists etc.  

Here is a link: [gist](https://gist.github.com/citrusui/07978f14b11adada364ff901e27c7f61)

**Comment by adityatelange** [comment, marked as answer]
That is beacause Hugo by default does not allow any HTML content to be rendered as it is. 

Either user the included `collapse` shortcode or search Hugo docs/Community to enable HTML rendering (something like unsafe=true)

**Reply by cs-mshah** [reply]
yep. I found it. thanks

**Reply by iAziz786** [reply]
@cs-mshah where have you found the solution?

**Reply by iAziz786** [reply]
Got it, this is how we should use it.

```md
{{< collapse summary="What is SPF?" >}}
Your content goes here. You can use **Markdown** or <strong>HTML</strong> inside the collapsible section.
{{< /collapse >}}
```
